website/docs/faq.md

```diff
@@ -4,4 +4,78 @@ title: FAQ
+That basically restricts portal installation along with its agent to a single namespace and the chaos operator, exporter all get installed in a single namespace and can only perform and monitor chaos in that namespace.
+Other than that there is another key in the control plane’s configmap `litmus-portal-admin-config` called  `AgentScope`, this is given to allow users to restrict access to the litmus self-agent components self-agent is the agent for your control plane cluster (exporter, operator etc), you can use both of them in a way to give access as per the requirement.
+The above holds for the control plane and self agent, for the external agents which can be connected using the litmusctl CLI you can provide the scope of the agent while using the utility to connect your other cluster to the control plane with access to just a single namespace or cluster-wide access.
+Using a combination of AgentScope: cluster and `PORTAL_SCOPE`  env set to cluster would give you cluster-admin privileges to inject chaos on all namespaces where the control plane/portal is installed. For external agents just selecting the scope of installation as cluster would be sufficient via litmusctl.
+
+### What does failed status of workflow means in LitmusPortal?
+
+Failed status indicates that either there is some misconfiguration in the workflow or  the default hypothesis of experiment was disproved and some of the experiments in the workflow failed, In such case the resiliency score will be less than 100.
+
+
+### How can I setup chaoshub of my own gitlab repo in Litmus Portal?
+
+In the litmus portal when you go to the chaoshub section and you click on connect new hub button, you can see that there are two modes of authentication i.e public mode and private mode. For public mode, you only have to provide the git URL and branch name.
+
+For  private mode, we have two types of authentication; Access token and SSH key.
+For the access token, go to the settings of GitLab and in the Access token section, add a token with read repository permission. After getting the token, go to the Litmus portal and provide the GitLab URL and branch name along with the access token. After submitting, your own chaos hub is connected to the Litmus portal.
+For the second mode of authentication i.e; SSH key, In SSH key once you click on the SSH, It will generate a public key. You have to use the public key and put it in the gitlab setting. Just go to the settings of gitlab, you can see the SSH key section, go to the SSH key section and add your public key. After adding the public key.Get the ssh type url  of the git repository and  put it in the Litmusportal along with the branch, after submitting  your own chaoshub is connected to the Litmus Portal.
```

Comment:
```suggestion
For the SSH key, click on the SSH and it will generate a public key. You have to use this public key and put it in your GitLab account. Just go to the settings of GitLab, you can see the SSH key section, go to the SSH key section and add your public key. After adding the public key. Get the ssh type URL of the git repository and put it in the Litmus Portal along with the branch. After submitting, your own chaoshub is connected to the Litmus Portal.
```